roles/certs/generate-csr/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+- name: "Generate the private Key and a CSR"
+  generate_csr:
```

Comment:
csr variables here are asking for a dictionary, but in the docs these are all separate variables

Reply:
done